elf/module.go

```diff
@@ -154,6 +166,14 @@ type TracepointProgram struct {
 	efd   int
 }
 
+// RawTracepointProgram represents a tracepoint program
+type RawTracepointProgram struct {
+	Name  string
+	insns *C.struct_bpf_insn
+	fd    int
+	efd   int
```

Comment:
Is `efd` used anywhere?

Reply:
no, removed it. 
Also added `closeRawTracepointPrograms` method